core/trino-main/src/main/java/io/trino/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -713,6 +722,26 @@ private Query showCreateSchema(ShowCreate node)
             return singleValueQuery("Create Schema", formatSql(createSchema).trim());
         }
 
+        private Node showCreateFunction(ShowCreate node)
+        {
+            QualifiedObjectName functionName = qualifiedFunctionName(functionSchema, node, node.getName());
+
+            accessControl.checkCanShowCreateFunction(session.toSecurityContext(), functionName);
+
+            Collection<LanguageFunction> functions = metadata.getLanguageFunctions(session, functionName);
+            if (functions.isEmpty()) {
+                throw semanticException(NOT_FOUND, node, "Function not found");
```

Comment:
nit: `"Function %s not found".formatted(functionName)`

Reply:
This matches what we do for other statements like `DROP FUNCTION`. This statement is for a specific function, so having the name doesn't seem to provide any value, and I think it's better to have a constant (non-variable) error message.